test/e2e/scenarios/README.md

```diff
@@ -0,0 +1,12 @@
+These e2e tests implement the use cases as described in this [Multicluster NetworkPolicy Proposal](https://docs.google.com/document/d/1_QzuasJPiQ-4t8tUGODoRS2E-Q3KLJajqMyCvLPH2tY/edit#heading=h.4igbcpajqich).
+
+Each test corresponds to each use case from the above mentioned proposal.
+
+To run these e2e tests, follow the steps as described [here](https://github.com/submariner-io/submariner#testing).
+
+There are two type of tests, dataplane and controlplane.
+Dataplane tests check for connectivity after network policies are created.
+Controlplane tests check for IP of target pod in IPBlocks on the cluster where the network policy is created.
```

Comment:
Controlplane tests check for IP of target pods in coastguard managed NetworkPolicies that instead of podSelectors use ipBlocks to reference remote pods.